pkg/services/folder/folderimpl/folder.go

```diff
@@ -74,37 +76,50 @@ func ProvideService(
 	cfg *setting.Cfg,
 	r prometheus.Registerer,
 	tracer tracing.Tracer,
-) *Service {
-	k8sHandler := &foldk8sHandler{
-		gvr:        v0alpha1.FolderResourceInfo.GroupVersionResource(),
-		namespacer: request.GetNamespaceMapper(cfg),
-		cfg:        cfg,
-	}
-
-	unifiedStore := ProvideUnifiedStore(cfg)
-
+	docs resource.DocumentBuilderSupplier,
```

Comment:
This isn't available to wire

Reply:
Unfortunately we can do that, `resource.ResourceClient` imports the `authz` client which in turn imports the Folder service.

This is the best option at the moment. I think we need to refactor the `resource` to provide a client that doesn't import  `authz`

Reply:
Yes the client could/should be able to just be interfaces

Reply:
also posted a message on [App Platform slack channel](https://raintank-corp.slack.com/archives/C04RVCAG9B5/p1737118715888179)